src/object.rs

```diff
@@ -758,6 +758,7 @@ macro_rules! glib_object_wrapper {
             }
 
             fn as_ptr(&self) -> *mut Self::GlibType {
+                use $crate::translate::ToGlibPtr;
                 self.0.to_glib_none().0 as *mut _
```

Comment:
```suggestion
                $crate::translate::ToGlibPtr::to_glib_none(&self.0).0 as *mut _
```

And the same below. Otherwise one could get conflicts if for some reason another trait with `to_glib_none()` was in scope :) Unlikely but easy to get around

Reply:
Good point; I hadn't though of such a conflict. Probably a good idea to prevent that, though it indeed seems unlikely.